eng/scripts/docs/Docs-ToC.ps1

```diff
@@ -155,3 +155,49 @@ function Parse-Overview-Frame ($filePath, $destination) {
     Add-Content -Path $destination -Value $namespaces
     Get-Content $destination
 }
+
+function Get-java-UpdatedDocsMsToc($toc) {
+    # Add "ADAL" to "Active Directory" service. This is onboarded through a repo
+    # source process that is not obvious in the CI configuration (no package
+    # name, only a repo URL)
+    $services = $toc[0].items
+    # Add service exsted in old toc.
+    $services += [PSCustomObject]@{
+        name  = "Active Directory"
+        href  = "~/docs-ref-services/{moniker}/resourcemanager-msi-readme.md"
+        landingPageType = "Service"
+        items = @(
+            [PSCustomObject]@{
+                name  = "Resource Management"
+                href  = "~/docs-ref-services/{moniker}/resourcemanager-msi-readme.md"
+                children = @("com.azure.resourcemanager.msi*")
+            }, 
+            [PSCustomObject]@{
+                name  = "client"
+                href  = "~/docs-ref-services/{moniker}/resourcemanager-msi-readme.md"
+                children = @(
+                    "com.microsoft.aad.adal*",
+                    "com.microsoft.aad.adal4j*",
+                    "com.microsoft.identity.client*")
+            })
+    }
+    $services += [PSCustomObject]@{
+        name  = "Edge Gateway"
+        landingPageType = "Service"
+        items = @(
+            [PSCustomObject]@{
+                name  = "Resource Management"
+                children = @("com.microsoft.azure.management.edgegateway*")
+            })
+    }
+    $services += [PSCustomObject]@{
+        name  = "Resource Mover"
+        landingPageType = "Service"
+        items = @(
+            [PSCustomObject]@{
+                name  = "Resource Management"
+                children = @("com.microsoft.azure.management.resourcemover.v2021_01_01*")
+            })
+    }
+    $services | Sort-Object -Property name
```

Comment:
This will result in `Other` being in the middle of the ToC instead of at the end. See here for how to sort and re-add other: https://github.com/Azure/azure-sdk-for-python/pull/23589/files#diff-4b590925689e1df66d5c28ddc67d15ded4db6d686eb7c7878e0c8238feb57e25R121

Reply:
Good catch. Addressed.

Reply:
Tested in local and passed.